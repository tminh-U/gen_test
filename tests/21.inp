8
1 2 2 12 9 3 4 15 
7
7 8
5 6
7 7
8 8
2 7
8 8
4 6
