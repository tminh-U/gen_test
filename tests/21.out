4
3
2
2
24
2
4
